lib/_stream_transform.js

```diff
@@ -67,8 +67,10 @@ function afterTransform(stream, er, data) {
 
   var cb = ts.writecb;
 
-  if (!cb)
-    return stream.emit('error', new Error('no writecb in Transform class'));
+  if (!cb) {
+    const err = new Error('no writecb in Transform class');
+    return process.nextTick(() => stream.emit('error', err));
+  }
```

Comment:
Same comment from above:

@mafintosh sorry for the ignorance, but why would error paths ever be hot?